src/tests/stdio/file_io_error_test.c

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file
+ * @brief Tests for feof(), ferror() and clearerr() in compat/libc/stdio/
+ *
+ * @author Suraj Upadhyay
+ * @date 08.03.2020
+ */
+
+#include <stdio.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("stdio/feof ferror clearerr test");
+
+#define TEST_FILE_NAME "/fileIOErrorTest.txt" 
```

Comment:
Are you sure the still works?

Reply:
Because you renamed the file to file_io_error_test.txt

Reply:
A lot of oopsies these days .... 
I will change it shortly :)